scripts/list_updatable_packages

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+from collections import namedtuple
+from pathlib import Path
+from typing import Generator, Iterable, Mapping, Optional
+
+from debian.changelog import Changelog
+from debian.debian_support import NativeVersion
+
+import requests
+
+SESSION = requests.Session()
+
+
+Package = namedtuple('Package', ['package_name', 'version', 'gem_name', 'directory'])
+
+
+def find_packages() -> Generator[Package, None, None]:
+    for plugin_rb in Path('plugins/').glob('**/*.rb'):
+        gem_name = plugin_rb.read_text().strip().split()[1].strip("',\"")
```

Comment:
Technically gem_name isn't really used anymore in the action so I wonder if we should drop this. In #7412 we could make the same argument. But I'm fine either way.

Reply:
It certainly is needed to ask `rubygems.org` about the version? :)